core/ibft.go

```diff
@@ -119,7 +114,6 @@ func NewIBFT(
 			},
 			seals:        make([]*messages.CommittedSeal, 0),
 			roundStarted: false,
-			name:         newRound,
```

Comment:
Perhaps declare the `commitSent: false,` here explicitly as well?

Reply:
Added 94978a3b28f7a8bef95d9fe77e78fea524c6c1eb